doc/api/prev_api_changes/api_changes_3.0.0.rst

```diff
@@ -106,7 +106,7 @@ Different exception types for undocumented options
 - Passing the undocumented ``xmin`` or ``xmax`` arguments to
   :meth:`~matplotlib.axes.Axes.set_xlim` would silently override the ``left``
   and ``right`` arguments.  :meth:`~matplotlib.axes.Axes.set_ylim` and the
-  3D equivalents (e.g. `~.Axes3D.set_zlim3d`) had a
+  3D equivalents (e.g. ``~.Axes3D.set_zlim3d``) had a
```

Comment:
Was this cross ref working before?  That so many of these are changing to code literal markup instead of xref markup makes me very concerned that something is going wrong.

Reply:
Yes, they were working before. The goal was to remove redundant aliased methods from the doc page. Removing aliased methods led to changing cross-references to literals.

Aliased Axes3D methods:
```
stem3D = stem
set_zlim3d = set_zlim
text3D = text
```

Excluding some deprecated methods (`w_xaxis`, `w_yaxis`, and `w_zaxis`) from the new Axes3D doc page led to changing their cross-references to literals as well.

Method example from axes3d.py:
```
w_xaxis = _api.deprecated("3.1", alternative="xaxis", removal="3.8")(
        property(lambda self: self.xaxis))
```

Reply:
I think Fixing these so that they are cross references again is going to be a condition for merging.

I would do two things here:

 - change these to point at the non-alaiased versions
 - add a section at the bottom of the page for "aliases and deprecated methods" and save making them literals until we actually remove them

Reply:
That makes sense, will do. I assumed editing historical method names in places like prev_api_changes would be frowned upon.

Reply:
Between changing links -> literals and changing the name to point to the non-alias, I think breaking the link is worse.